.pipelines/.vsts-vhd-builder.yaml

```diff
@@ -191,6 +191,7 @@ stages:
             if [[ ${OS_VERSION} == "22.04" && ${ARCHITECTURE,,} == "arm64" ]]; then SKU_NAME="${SKU_NAME}arm64"; fi && \
             if [[ ${OS_VERSION} == "18.04" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
             if [[ ${OS_VERSION} == "20.04" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
+            if [[ ${OS_VERSION} == "V2" && ${OS_SKU} == "CBLMariner" && ${ENABLE_FIPS,,} == "true" ]]; then SKU_NAME="${SKU_NAME}fips"; fi && \
```

Comment:
Then the Definition in sig_config.go in your previous PR should be changed to V2gen2fips
![image](https://user-images.githubusercontent.com/39974812/236064300-74bbb13d-e927-494d-be31-fef7d0519d80.png)